.github/workflows/ci.yml

```diff
@@ -103,6 +104,7 @@ jobs:
         with:
           repository: 'oceanprotocol/barge'
           path: 'barge'
+          ref: feature/bump_contracts_and_subgraph
```

Comment:
we are going to remove this once the barge pr is merge right ?

Reply:
yes